### PR TITLE
Initialize with essential routes

### PR DESCRIPTION
**Pull request by itexpert120**
Resolves https://github.com/NEARBuilders/gateway/issues/274

### CODE REVIEW

File: apps/new/widget/Index.jsx [hunk 1]
```diff
@@ -1 +1,30 @@
...
+const config = {
+  theme: {},
+  layout: {
+    src: "${alias_devs}/widget/Layout",
+    props: {
+      variant: "standard",
+    },
+  },
+  blocks: {
+    Header: () => <></>,
```
Comment (elliotBraem) [inline apps/new/widget/Index.jsx:10]: Can you initialize with Header as the navbar? Let's go ahead and pull over the right side too

image

Let's not pass in router object to it this time

Comment (elliotBraem) [reply apps/new/widget/Index.jsx:10]: And grab the footer from the landing page

components.header and components.footer? navbar? What do you think, here is the [link to canvas](https://www.tldraw.com/r/rw9W7SsJnejDFsWFznT5K?v=-902,-1781,14400,7940&p=Q2lUnVRyH5IoXJftWXx40)

Comment (itexpert120) [reply apps/new/widget/Index.jsx]: header and footer makes sense

Comment (itexpert120) [reply apps/new/widget/Index.jsx]: we will fork the navbar as header later when we need to make changes

**Pull request merged**